### DISCUSSION TITLE
Mind The Gap Funniest Joke Competition

### DISCUSSION THREAD

**Discussion by therealkujo** [mindthegap-workshop/mindthegap-workshop.github.io#2] [Announcements]
Welcome to the Mind The Gap Funniest Joke Competition! Each team will have their team lead submit their *funniest* joke and everybody will have the opportunity to vote for the funniest joke! 

We will go on a trust based system for voting and we ask that you limit your votes to 5 of your favourite jokes! The team with the most votes wins!

To make a submission, simply create a comment with the following format:

```
Team TEAM_NAME: 
Joke:
FUNNY JOKE HERE!
```

**Comment by masindeg** [comment]
## From GIT-CAT
Eight bytes walk into a bar. The bartender asks, “Can I get you anything?” “Yeah,” reply the bytes. “Make us a double.

**Reply by megamanics** [reply]
🚀 🚀 🚀 🚀 🚀

**Reply by NosiphoDhladhla** [reply]
**From Cat_Wrangling**

How does a farmer get a girlfriend? He a tractor!

**Reply by megamanics** [reply]
Why did the programmer quit his job? didn't get arrays.

**Comment by Himmelsgucker1981** [comment]
From Germny_Jokes

Why did the German sausage feel self-conscious? Because it was afraid it would be the wurst.
